Sources/TuistKit/Utils/CommentsRemover.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+enum CommentsRemover {
+    static func removeComments(from code: String) throws -> String {
+        let regexPattern = #"//.*?$|/\*[\s\S]*?\*/"#
+        do {
+            let regex = try NSRegularExpression(pattern: regexPattern, options: [.anchorsMatchLines])
+            let range = NSRange(location: 0, length: code.utf16.count)
```

Comment:
```suggestion
            let range = NSRange(location: 0, length: code.count)
```

Do we really need to go through `utf16`?